backend/server/static/api/meetingbook.yaml

```diff
@@ -163,22 +164,230 @@ paths:
+    summary: endpoint for the settings of the booking, controlled by admin.
+    get:
+      summary: get current settings
+      parameters:
+      responses:
+        "200":
+          description: "Successful operation"
+          schema:
+            $ref: "#/definitions/BookingSettings"
+        "400":
+          description: "Bad Request"
+        "401":
+          description: "Unauthorized"
+      security:
+        - basic_auth: []
+    put:
+      summary: Update booking settings
+      parameters:
+        - in: body
+          name: interval_id
+          summary: description of the new booking settings
+          schema:
+            $ref: "#/definitions/BookingSettings"
+      responses:
+        "200":
+          description: "Successfully updated"
+          schema:
+            $ref: "#/definitions/BookingSettings"
+        "400":
+          description: "Bad Request"
+        "401":
+          description: "Unauthorized"
+      security:
+        - basic_auth: []
+    post:
+      summary: Add a new interval
+      description: |
+            request to add a new free interval for the admin. Format:
+            `/schedule/start=2021-02-02T10:00&end=2021-02-02T14:00`
+
+      parameters:
+        - in: query
+          name: start
+          type: string
+          required: true
+          description: |
+            Start interval in format: `start=2021-02-02T10:00`
+        - in: query
+          name: end
+          type: string
+          required: true
+          description: |
+            End interval in format: `end=2021-02-02T10:00`
+      responses:
+        "200":
+          description: "Successful operation"
+          schema:
+            $ref: "#/definitions/Slot"
+        "400":
+          description: "Bad Request"
+        "401":
+          description: "Unauthorized"
+      security:
+        - basic_auth: []
+  /calendars/{link_id}:
+    get:
+      summary: |
+        Returns a calendar according to the shareable link id. This is an endpoint for Guest.
+        Unprotected endpoint.
+      parameters:
+        - in: path
+          name: link_id
+          type: string
+          required: true
+          description: |
+              identifier of the shareable link (public calendar). See  "#/definitions/CalendarLink" for the format details.
+      responses:
+        "200":
+          description: "Successful operation"
+          schema:
+            $ref: "#/definitions/Calendar"
+        "400":
+          description: "Bad Request"
+          schema:
+            $ref: "#/definitions/Problem"
+        "401":
+          description: "Unauthorized - link has expired"
+          schema:
+            $ref: "#/definitions/Problem"
+        "404":
+          description: "Shareable link not found"
+          schema:
+            $ref: "#/definitions/Problem"
+    delete:
+      summary: Delete shareable link (public calendar) for the admin
+      parameters:
+        - in: path
+          name: id
```

Comment:
error in swagger: "Path parameter "id" must have the corresponding {id} segment in the "/calendars/{link_id}" path
maybe name: link_id?

Reply:
```suggestion
          name: link_id
```

Reply:
Ack